lib/cc/config/yaml/validator.rb

```diff
@@ -48,10 +48,19 @@ def invalid_engine_warnings
 
         def invalid_engines
           return [] unless cc_yaml_processable?
-          config = CC::Config::YAML.new(path)
-          config.engines.reject do |engine|
-            engine_exists?(engine)
-          end
+          config = CC::Config::YAMLAdapter.load(path)
+          config.config["plugins"]
+            .map do |name, data|
+              Engine.new(
+                name,
+                enabled: data.fetch("enabled", true),
+                channel: data["channel"],
+                config: data["config"],
```

Comment:
Do `channel` and `config` account for `nil` values correctly within `Engine`? Or do we need `stable` and `{}` fallbacks here?

Reply:
Both of those are defaulted as expected within `Engine`.